includes/admin/class-sensei-onboarding.php

```diff
@@ -385,4 +386,112 @@ public function update_wizard_user_data( $changes ) {
 		$option = array_merge( $this->get_wizard_user_data(), $changes );
 		return update_option( self::USER_DATA_OPTION, $option );
 	}
+
+	/**
+	 * Override sensei extensions.
+	 *
+	 * @param array $original_extensions Original extensions.
+	 *
+	 * @return array Overridden extensions.
+	 */
+	private function override_sensei_extensions( $original_extensions ) {
+		$override_extensions = [
+			'sensei-course-progress' => [
+				'title'  => __( 'Course progress', 'sensei-lms' ),
+				'status' => 'installed',
+			],
+			'sensei-certificates'    => [
+				'title'  => __( 'Certificates', 'sensei-lms' ),
+				'status' => 'error',
+			],
+			'sensei-wc-paid-courses' => [
+				'title'                        => __( 'WooCommerce Paid Courses', 'sensei-lms' ),
+				'confirmationExtraDescription' => __(
+					'(The WooCommerce plugin may also be installed and activated for free.)',
+					'sensei-lms'
+				),
+				'status'                       => 'loading',
+			],
+		];
+
+		return array_map(
+			function( $extension ) use ( $override_extensions ) {
+				// Decode price.
+				if ( isset( $extension['price'] ) && 0 !== $extension['price'] ) {
+					$extension['price'] = html_entity_decode( $extension['price'] );
+				}
+
+				$extension_id = $extension['id'];
+
+				if ( ! isset( $override_extensions[ $extension_id ] ) ) {
+					return $extension;
+				}
+
+				foreach ( $override_extensions[ $extension_id ] as $key => $value ) {
+					$extension[ $key ] = $value;
+				}
+
+				return $extension;
+			},
+			$original_extensions
+		);
+	}
+
+	/**
+	 * Normalize sensei extensions array.
+	 *
+	 * @param array $raw_extensions Raw extensions.
+	 *
+	 * @return array Normalized extensions.
+	 */
+	private function normalize_sensei_extensions( $raw_extensions ) {
+		$json = json_decode( $raw_extensions['body'], true );
+
+		// Normalize property names.
+		$properties = [
+			'product_slug' => 'id',
+			'excerpt'      => 'description',
+			'link'         => 'learnMoreLink',
```

Comment:
Instead of changing the property names, WDYT about changing them on the frontend to use the original names? Should be simpler if it's not too much work.

Reply:
Looks good! I just added the `slug` prop, so we don't need to normalize it in the frontend, because using with `_` doesn't match with our lint rules. https://github.com/Automattic/sensei/pull/3176/commits/e1ba40aae4777e79414ecdc024f73a4432dde6d0

Reply:
I refactored again to normalize it in the frontend, because I had to normalize also the feature titles ;)
https://github.com/Automattic/sensei/pull/3176/commits/42e54d6851e7638f2bf0dabc7a4581a7d29e73a9
https://github.com/Automattic/sensei/pull/3176/commits/5b56e6613f3acb21314722ad3a2cb04dd159e61b